scripts/copy_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mysqldump -h codingmanager.com -u admin -p${CODING_MANAGER_PWD} codingmanager  > codingmanager_backup.sql
```

Comment:
Please also use an env var for hostname/ip, db user, db name. For example,
```suggestion
mysqldump -h {CODING_MANAGER_DB_IP} -u {CODING_MANAGER_USER} -p${CODING_MANAGER_PWD} {CODING_MANAGER_DB}  > codingmanager_backup.sql
```